consensus/state.go

```diff
@@ -739,12 +767,13 @@ func (cs *ConsensusState) EnterPrevoteWait(height int, round int) {
 // Enter: any +2/3 precommits for next round.
 // Lock & precommit the ProposalBlock if we have enough prevotes for it,
 // else, unlock an existing lock and precommit nil if +2/3 of prevotes were nil,
-// else, precommit locked block or nil otherwise.
+// else, precommit nil otherwise.
+// NOTE: we don't precommit our locked block (unless theres another POL for it) because it complicates unlocking and accountability
```

Comment:
oh right it shouldn't say "complicates" things but rather "breaks things".

though on the other hand, if we enforce passing polkas with precommits then i think it can work, but just much more complicated